osu.Game.Tests/Visual/UserInterface/TestSceneModSettings.cs

```diff
@@ -25,6 +25,8 @@ public class TestSceneModSettings : OsuTestScene
 
         private readonly Mod testCustomisableMod = new TestModCustomisable1();
 
+        private readonly Mod testCustomisableAutoOpenMod = new TestModCustomisable3();
```

Comment:
```suggestion
        private readonly Mod testCustomisableAutoOpenMod = new TestModCustomisable2();
```

Reply:
Thank you, but I'm confused, why? `TestModCustomisable2` doesn't override `RequiresConfiguration`. Should I remove Customisable3 and override property in Customisable2 instead?

Reply:
Currently `TestModCustomizable2` has no use, so I proposed that.